### ISSUE TITLE
[BUG] enabling all cores from solr-example-all-languages.xml produces solr error

### ISSUE THREAD

**Issue by timoschmidt** [TYPO3-Solr/ext-solr#119]
When i copy solr-example-all-language-xml and the core configuration to a solr server with version 4.10.4 and the solr plugin i get the following error:

SolrCore Initialization Failures

    core_my: org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core core_my: Plugin init failure for [schema.xml] fieldType "text": Plugin init failure for [schema.xml] analyzer/tokenizer: Error loading class 'solr.ICUTokenizerFactory'. Schema file is /var/solr/typo3cores/conf/burmese/schema.xml
    core_km: org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core core_km: Plugin init failure for [schema.xml] fieldType "text": Plugin init failure for [schema.xml] analyzer/tokenizer: Error loading class 'solr.ICUTokenizerFactory'. Schema file is /var/solr/typo3cores/conf/khmer/schema.xml
    core_lo: org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core core_lo: Plugin init failure for [schema.xml] fieldType "text": Plugin init failure for [schema.xml] analyzer/tokenizer: Error loading class 'solr.ICUTokenizerFactory'. Schema file is /var/solr/typo3cores/conf/lao/schema.xml
    core_pl: org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core core_pl: Plugin init failure for [schema.xml] fieldType "text": Plugin init failure for [schema.xml] analyzer/filter: Error loading class 'solr.StempelPolishStemFilterFactory'. Schema file is /var/solr/typo3cores/conf/polish/schema.xml

**Comment by timoschmidt** [comment]
Sorry, can be closed, my fault. The contrib and dist folder needs to be copied to the root folder and then everything works fine!

**Comment by ervaude** [comment]
Hi, I have encountered this issue right know. Do you mind telling me which folders you were talking about (contrib, dis, and root)? I seem not to be able to figure it out myself at the moment. Thank you very much.

**Comment by timoschmidt** [comment]
@ervaude 

You need to copy the contrib and dist folder (shipped with solr 4.10.4) beside the "conf" folder.
You should be save by using "Resources/Install/install-solr-tomcat.sh" this is doing the same:

```bash
# copy libs
cd /opt/solr-tomcat/
cp -r solr-$SOLR_VERSION/dist solr/
cp -r solr-$SOLR_VERSION/contrib solr/
```
